modules/cms/lb.tf

```diff
@@ -124,26 +124,12 @@ resource "aws_security_group" "lb_ingress" {
 
   # For Web traffic to services
   ingress {
-    cidr_blocks = ["0.0.0.0/0"]
-    protocol    = "tcp"
-    from_port   = 80
```

Comment:
I think this is safe, but am not sure if circle relies on any type of forwarding behavior